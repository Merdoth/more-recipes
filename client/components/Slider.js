import React from 'react';

export default () => {
  return (
      <div className="hero-bg">
        <div className="container-fluid">
          <div id="carouselExampleSlidesOnly" className="carousel slide" data-ride="carousel" >
            <div className="carousel-inner" role="listbox" style={{"height":550+'px'}}>
              <div className="carousel-item active">
                <img className="d-block img-fluid " style={{"width":100+'%',"height":550+'px'}} src={require('../image/Salad.jpg')} alt="First slide"/>
                <div className="hello">
                <h1>More Recipes</h1>
                <h5><i>Your Favorite Recipe's At Your Pleasure.</i></h5>
                </div>
              </div>
              <div className="carousel-item " >
                <img className="d-block img-fluid" style={{"width":100+'%',"height":550+'px'}} src={require("../image/Food.jpg")} alt="Second slide"/>
                <div className="hello">
                <h1>Welcome</h1>
                <h5><i>Feel Free To Explore Your Ideas.</i></h5>
                </div>
              </div>
              <div className="carousel-item">
                <img className="d-block img-fluid" style={{"width":100+'%',"height":550+'px'}} src={require("../image/shrimp.jpg")} alt="Third slide"/>
                <div className="hello">
                <h1>Build Your Recipes</h1>
                <h5><i>Love Food, Love Life.</i></h5>
                </div>
            </div>
            </div>
          </div>
        </div>
      </div>
  )

}