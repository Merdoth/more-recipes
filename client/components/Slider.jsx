import React from 'react';
/**
 * @description this returns a slider component
 *
 * @param { Function } Slider
 *
 * @returns { Slider } Slider
 */

const Slider = () => (
  <div className="hero-bg">
    <div className="container-fluid">
      <div
        id="carouselExampleSlidesOnly"
        className="carousel slide"
        data-ride="carousel"
      >
        <div className="carousel-inner" role="listbox">
          <div
            className="carousel-item active"
            style={{ backgroundImage: "url('/assets/image/Salad.jpg')" }}
          >
            <div className="slider-text">
              <h5 className="hero-header">
                Your Favorite Recipe's At Your Pleasure.
              </h5>
            </div>
          </div>
          <div
            className="carousel-item"
            style={{ backgroundImage: "url('/assets/image/Food.jpg')" }}
          >
            <div className="slider-text">
              <h5 className="hero-header">Feel Free To Explore Your Ideas.</h5>
            </div>
          </div>
          <div
            className="carousel-item"
            style={{ backgroundImage: "url('/assets/image/shrimp.jpg')" }}
          >
            <div className="slider-text">
              <h5 className="hero-header">Love Food, Love Life.</h5>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
);

export default Slider;
