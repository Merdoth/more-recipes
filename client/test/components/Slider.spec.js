import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { Slider } from '../../components/Slider.jsx';

const setup = () => shallow(<Slider/>);

describe('Slider Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
