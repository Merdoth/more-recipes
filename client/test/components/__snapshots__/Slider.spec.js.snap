// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider Component snapshot it should render the right amount of elements 1`] = `
<div
  className="hero-bg"
>
  <div
    className="container-fluid"
  >
    <div
      className="carousel slide"
      data-ride="carousel"
      id="carouselExampleSlidesOnly"
    >
      <div
        className="carousel-inner"
        role="listbox"
      >
        <div
          className="carousel-item active"
          style={
            Object {
              "backgroundImage": "url('/assets/image/Salad.jpg')",
            }
          }
        >
          <div
            className="slider-text"
          >
            <h5
              className="hero-header"
              id="slider-one"
            >
              Your Favorite Recipe's At Your Pleasure.
            </h5>
          </div>
        </div>
        <div
          className="carousel-item"
          style={
            Object {
              "backgroundImage": "url('/assets/image/Food.jpg')",
            }
          }
        >
          <div
            className="slider-text"
          >
            <h5
              className="hero-header"
            >
              Feel Free To Explore Your Ideas.
            </h5>
          </div>
        </div>
        <div
          className="carousel-item"
          style={
            Object {
              "backgroundImage": "url('/assets/image/shrimp.jpg')",
            }
          }
        >
          <div
            className="slider-text"
          >
            <h5
              className="hero-header"
            >
              Love Food, Love Life.
            </h5>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
